### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by vldtsw**

### CODE REVIEW

**Review by Oleksandr563487953 — APPROVED** [review]

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,36 @@
...
+    def serve_cars(self, cars_list: list):
+        income = sum([self.calculate_washing_price(car) for car in cars_list
+                      if car.clean_mark < self.clean_power])
+        for car in cars_list:
+            self.wash_single_car(car)
+        return income
+
+    def calculate_washing_price(self, car: Car):
+        clean_dif = self.clean_power - car.clean_mark
+        clean_ratio = self.average_rating / self.distance_from_city_center
```
Comment (MasterpieceElbow) [inline app/main.py:26]: There is not sense to make individual variable

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (Oleksandr563487953) [reply app/main.py:26]: I thought it is more clean to understand, so i approved it.

Comment (vldtsw) [reply app/main.py:26]: I made it to avoid "E501 line too long" :(

**Review by IvanRamyk — APPROVED** [review]
Good job!

**Pull request closed without merging**